docs/react/guides/installing-tailwindcss-in-web-react-nextjs-apps.md

```diff
@@ -0,0 +1,71 @@
+# Installing TailwindCss in Web/React/Next.js Apps
+
+This guide will serve as a quickstart to installing TailwindCss in your Web/React/Next.js app.
```

Comment:
Should be `Tailwind CSS` not `TailwindCss`. Maybe also link out to `https://tailwindcss.com/`?